src/lib/components/TocScrollSpy/index.js

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the Licenspe.
+ */
+
+import {ScrollSpy} from '../ScrollSpy';
+
+/**
+ * Element that adds active classes to its children when the user scrolls to
+ * a heading this child points to.
+ * @extends {ScrollSpy}
+ * @final
+ */
+class TocScrollSpy extends ScrollSpy {
+  constructor() {
+    super();
+    this.selector = this.selector = 'h2[id], h3[id]';
+    this.tocActiveClass = 'w-scroll-spy__active';
```

Comment:
This and the following property act as [static class properties](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/static), and would be cleaner if defined as such, instead of being set as instance properties post-construction.